src/tests/p15dump.c

```diff
@@ -66,10 +66,11 @@ static int dump_unusedspace(void)
 
 	if (p15card->file_unusedspace != NULL)
 		path = p15card->file_unusedspace->path;
-	else {
+	else if (p15card->file_app != NULL) {
 		path = p15card->file_app->path;
 		sc_append_path_id(&path, (const u8 *) "\x50\x33", 2);
-	}
+	} else
+		return -1;
```

Comment:
(sorry for the late reply. I missed this comment somehow)
@frankmorgner I am not sure what you mean by the relative path in this context.
Writing some error would make sense. Thanks for the idea.